modules/porous_flow/src/materials/PorousFlowTemperature.C

```diff
@@ -15,7 +15,7 @@ InputParameters
 PorousFlowTemperature::validParams()
 {
   InputParameters params = PorousFlowMaterial::validParams();
-  params.addCoupledVar("temperature", 20.0, "Fluid temperature variable");
+  params.addCoupledVar("temperature", 293.0, "Fluid temperature variable");
```

Comment:
Should we note the default in the description here?

Reply:
Yep, good idea